client/ayon_houdini/plugins/publish/extract_active_view_thumbnail.py

```diff
@@ -42,6 +44,21 @@ def process(self, instance):
                        .format(view_thumbnail)
         )
         instance.data["thumbnailPath"] = view_thumbnail
+      
+        instance.data.setdefault("representations", [])
+        representation = {
+            "name": "thumbnail",
+            "ext": "jpg",
+            "files": os.path.basename(view_thumbnail),
+            "stagingDir": os.path.dirname(view_thumbnail),
+            "thumbnail": True
+        }
+
+        # Adding representation to instance
+        instance.data["representations"].append(representation)
+        self.log.debug(
+            "Adding thumbnail representation: {}".format(representation)
+        )
```

Comment:
```suggestion
```

you shouldn't add representation for the thumbnail as there are dedicated plugins for creating the representation. 
- [extract_thumbnail_from_source.py](https://github.com/ynput/ayon-core/blob/develop/client/ayon_core/plugins/publish/extract_thumbnail_from_source.py)
- [extract_thumbnail.py](https://github.com/ynput/ayon-core/blob/develop/client/ayon_core/plugins/publish/extract_thumbnail.py)

Reply:
yes! 
Houdini host is missing as you have pointed out.